src/Compositor.cpp

```diff
@@ -2389,13 +2389,25 @@ void CCompositor::scheduleFrameForMonitor(CMonitor* pMonitor) {
 }
 
 CWindow* CCompositor::getWindowByRegex(const std::string& regexp) {
+    if (regexp == "activewindow") {
```

Comment:
Removed {}, oops!

I'd like to reiterate on the other proposals though, since I'm not confident enough it'd be a good change. While options are great to have, I would consider these ones an unneeded clutter.
1) Aliases can be neat in some cases (e.g. color vs colour / --list vs -l), but having *that much* different things doing the same thing is not IMHO. Opinionated, but makes sharing things more cumbersome. Someone could share their rules, and you'd have to remember to change them to your liking or end up with a mixed config, while there could simply be one entry that everyone uses.
2) Pretty much the same as 1), but also allows for crazy stuff like `activewindowRand0mStuuf`, which I'm not sure when would be actually preferred over the explicit rule.

As for the current naming, I went with `activewindow` initially to be consistent with the hyprctl's activewindow dispatcher (I think it's quite intuitive).
Will add proposals if preferred, though